src/marqo/client.py

```diff
@@ -232,59 +236,76 @@ def health(self):
         ", instead use 'client.index(index_name).eject_model()"
     )
     def eject_model(self, model_name: str, model_device: str):
+        if isinstance(self.config.instance_mapping, MarqoCloudInstanceMappings):
+            self.raise_error_for_cloud()
         return self.http.delete(path=f"models?model_name={model_name}&model_device={model_device}")
 
     @deprecated(
         "This method is deprecated and will be removed in Marqo 2.0.0"
         ", instead use 'client.index(index_name).get_loaded_models()"
     )
     def get_loaded_models(self):
+        if isinstance(self.config.instance_mapping, MarqoCloudInstanceMappings):
+            self.raise_error_for_cloud()
         return self.http.get(path="models")
 
     @deprecated(
```

Comment:
This annotation seems to support `reason` and `version`. We should use those. https://pypi.org/project/Deprecated/